endpoints/openrtb2/amp_auction.go

```diff
@@ -143,16 +143,18 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 	defer cancel()
 
 	usersyncs := usersync.ParsePBSCookieFromRequest(r, &(deps.cfg.HostCookie))
-	if req.App != nil {
-		labels.Source = pbsmetrics.DemandApp
+	labels.Source = pbsmetrics.DemandWeb
```

Comment:
Is there a reason why we're assuming the request to be of type `web` always and not considering the possibility of it being an `app`?

Reply:
This endpoint does not support Apps, therefore we always expect `req.App` to be `NULL`